Classes/Domain/Transformer/FileTransformer.php

```diff
@@ -5,21 +5,22 @@
 namespace DFAU\ToujouApi\Domain\Transformer;
 
 use DFAU\ToujouApi\Domain\Repository\FileRepository;
+use DFAU\ToujouApi\Transformer\ResourceTransformerInterface;
 use League\Fractal\TransformerAbstract;
 
-class FileTransformer extends TransformerAbstract
+class FileTransformer extends TransformerAbstract implements ResourceTransformerInterface
 {
-    public function transform(array $file): array
+    public function transform($data): array
```

Comment:
why no type hint?

Reply:
the method signature must match `ResourceTransformerInterface`